cmd/cluster/context.go

```diff
@@ -601,28 +619,38 @@ func GetPagerdutyClient(usertoken string, oauthtoken string) (*pd.Client, error)
 	if client != nil {
 		return client, err
 	}
+
 	client, err = getPDOauthClient(oauthtoken)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create both user and oauth clients for pd")
+		return nil, fmt.Errorf("failed to create both user and oauth clients for pd, neither key pd_oauth_token or pd_user_token are set in config file")
 	}
 	return client, err
 }
 
-func GetPDSeviceID(baseDomain string, usertoken string, oauthtoken string) (string, error) {
+func GetPDSeviceID(baseDomain string, usertoken string, oauthtoken string, team_ids []string) ([]string, error) {
```

Comment:
```suggestion
func GetPDServiceID(baseDomain string, usertoken string, oauthtoken string, team_ids []string) ([]string, error) {
```

Small nit :D Also where this function gets called in this file